esrc/esrcoordadmin.php

```diff
@@ -244,22 +244,22 @@ function displayLine($row, $charname, $finished, $system, $notes, $isCoord, $sum
 	// create object instances
 	//$users = new Users($database);
 	$rescue = new Rescue($database);
-	
+
 	$status = $row['status'];
 	$colspan = 0;
-	
+
 	echo "<tr>";
-	
+
 	// "Update" button - display only for finished requests if coord logged in
 	if (($isCoord == 1 && $finished == 1) || ($finished == 0 && $noUpdate == 0)) {
-		echo '<td><a type="button" class="btn btn-danger" role="button" href="?sys='.
+		echo '<td><a type="button" class="btn btn-danger" role="button" href="rescueoverview.php?sys='.
```

Comment:
Let's see what Igaze says about this - clearly it didn't work before. Because the modal isn't even loaded with this page...
I am surprised there isnt an issue on this, maybe there is.,  I didnt look.

Reply:
Igaze did comment positively, so I guess it is all good.